### PR TITLE
Linkadded

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

fixes issue #264

## Changes proposed

add-sub-category css

## Screenshots

Screenshot (741)

## Note to reviewers
now I have fix the issue you can review the PR.

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hi, thanks for rising the PR. could you please add this category in YouTube section?

File: database/frontend/css.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Kevin Powell",
+        "description": "It will give you idea about css. How to use css and many more things.",
+        "url": "https://www.youtube.com/@KevinPowell",
+        "category": "frontend",
+        "subcategory": "css",
+        "language": "english"
```
Comment (CBID2) [inline database/frontend/css.json:8]: Add a comma after "english".

Comment (rupali-codes) [reply database/frontend/css.json:8]: No, it will cause errors

**Review by CBID2 — APPROVED** [review]
LGTM! 👍🏾

File: database/frontend/css.json (hunk 1 above)
Comment (parma1111) [reply database/frontend/css.json:8]: Ok I will change

**Comment by parma1111** [comment]
now I updated in youtube css category
Screenshot (742)

**Review by rupali-codes — APPROVED** [review]
thanks for the changes, lgtm

**Pull request merged**